packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,400 @@
+system font for Apple platforms. We represent native web element styles (like option lists in <Anchor href="/form-elements/select/">Select</Anchor>) in
+our design assets as macOS styles.
+
+#### Whitney
+
+If you are using the Console theme in Paste, you will need to download Whitney.
+Reach out to us in `#help-design-system` if you need access to the Whitney font
+files for your work.
+
+## How to use Sketch libraries
+
+For every Abstract project that you work on where you plan on using Paste, you
+will need to link the necessary Paste Sketch libraries in order to use the
+Sketch symbols, text styles, and layer styles.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Tips on linking libraries in Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you haven’t linked libraries in Abstract before,
+    follow <Anchor href="https://help.abstract.com/libraries/link-libraries/">this guide</Anchor> to learn how.
+  </CalloutText>
+</Callout>
+
+### Paste design tokens
+
+<Flex>
+  <Tooltip text="There is ongoing maintenance and net new work in this project. Libraries in this project are at parity with code.">
+    <Anchor><SuccessIcon decorative={false} title="Success icon" color="colorTextSuccess"/></Anchor>
+  </Tooltip>
+  <Text as="p" marginBottom="space50" marginLeft="space20">
+    Libraries in this project are actively maintained
+  </Text>
+</Flex>
+
+The `Paste design tokens` project houses the text and layer styles
+for each theme in Paste. These libraries represent
+the <Anchor href="/tokens">tokens</Anchor> available for use in the system. For
+example, in the `default theme` file, you can find layer styles
+for every background color token that is available for the Default theme.
+
+#### Linking the library
+
+There are two libraries available in this project: `console theme` (this is the current
+theme you see in Console) and `default theme` (this is the new unified theme, formerly known as the "Unified Design Language" or "UDL").
+
+Link whichever library matches the theme you are using in your project. For example, if you are designing using the
+Default theme, link the `default theme` library to your project.
+
+![Modal in Abstract where you can select libraries from the Paste design tokens project to link to your project](../../../assets/images/design-guidelines/link-theme-library-modal.png)
+
+<Callout variant="warning">
+  <CalloutTitle>
+    A note on the SendGrid theme
+  </CalloutTitle>
+  <CalloutText>
+    You will also see the SendGrid theme in this project, but this theme is not actively maintained due to low usage.
+    If you plan to use this theme, please reach out to us in #help-design-system.
+  </CalloutText>
+</Callout>
+
+#### Using the library
+
+Once you have this library linked, you will see text styles (which include text colors, font families, and font sizes) and layer
```

Comment:
this library or these libraries?

Reply:
This library - you should really only ever need to link one theme library to a given project.